### DISCUSSION TITLE
End user feedbacks and results

### DISCUSSION THREAD

**Discussion by pl-marasco** [esa-cdab/cdab-testsuite#9] [General]
It seems that the entire system isn't built to provide users with results. It would be useful to create a way to push results to a common and open repository so that users will have the possibility to make comparisons with other results. 

Another point that pop up in my mind is that there is no information harvested regarding the connection ping and TTL. Am I wrong ?

**Comment by esa-cdab** [comment]
Hello pl-marasco,
yes currently there is no way to push results to a common and open repository.
Anyway, if you want compare your results with a large statistics results you can use the CDAB service public dashboard:
https://cdab-dashboard.exprivia.it/en-US/account/login
accessing with guest account benchmark / benchmark.
You can find more info on the CDAB service at this public repository:
https://github.com/esa-cdab/cdab-service

There, you will find also the service design document:
https://github.com/esa-cdab/cdab-service/blob/main/docs/Service_Design_Document_V2_2_1b_signed.pdf
that will answer to your second point also (Annex E, M001).
